examples/ipv4-prefix-delegation/variables.tf

```diff
@@ -0,0 +1,5 @@
+variable "deploy_sample_app" {
+  default     = false
+  type        = bool
+  description = "Deploy a sample app"
+}
```

Comment:
This variable is not used anywhere

Reply:
thanks for catching that - looks like I had some leftover copy+pasta to clean up!